ORA.Tracker/Services/Databases/IClusterDatabase.cs

```diff
@@ -0,0 +1,16 @@
+using ORA.Tracker.Models;
+
+namespace ORA.Tracker.Services.Databases
+{
+    public interface IClusterDatabase
```

Comment:
Rename to IDatabase instead of IClusterDatabase so that it can be reusable for other types